rep-2000.rst

```diff
@@ -418,6 +418,112 @@ Build System Support:
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minumum language requirements:
+
+- C++14
+- Python 3.5
+
+
+Dependency Requirements:
+
++-------------+-------------------------------------------------+---------------------+
+|             |                        Required Support         | Recommended Support |
++-------------+----------------+---------------+----------------+---------------------+
+| Package     | Ubuntu  Bionic |     MacOS**   |   Windows 10** | Debian Buster       |
++=============+================+===============+================+=====================+
+| CMake       |     3.10.2     |     3.14.4    |     3.14.4     |        3.13.4       |
++-------------+----------------+---------------+----------------+---------------------+
+| EmPY        |     3.3.2      |     3.3.2     |     3.3.2      |        3.3.2        |
++-------------+----------------+---------------+----------------+---------------------+
+| Gazebo      |     9.0.0      |     9.9.0     |      N/A       |        9.8.0*       |
++-------------+----------------+---------------+----------------+---------------------+
+| Ogre        |                                      1.10*                            |
++-------------+----------------+---------------+----------------+---------------------+
+| OpenCV      |     3.2.0      |     4.1.0     |     3.4.6*     |        3.2.0        |
++-------------+----------------+---------------+----------------+---------------------+
+| OpenSSL     |     1.1.0g     |     1.0.2r    |     1.0.2r     |        1.1.1c       |
++-------------+----------------+---------------+----------------+---------------------+
+| Poco        |     1.8.0      |     1.9.0     |     1.8.0*     |        1.9.0        |
++-------------+----------------+---------------+----------------+---------------------+
+| Python      |     3.6.5      |     3.7.3     |     3.7.3      |        3.7.3        |
++-------------+----------------+---------------+----------------+---------------------+
+| Qt          |     5.9.5      |     5.12.3    |     5.10.0     |        5.11.3       |
++-------------+----------------+---------------+----------------+---------------------+
+|                              |         **Linux only**                               |
++-------------+----------------+---------------+----------------+---------------------+
+| PCL         |     1.8.1      |     N/A       |     N/A        |        1.9.1        |
++-------------+----------------+---------------+----------------+---------------------+
+|                                **RMW DDS Milddleware Providers**                    |
++-------------+----------------+---------------+----------------+---------------------+
+| Connext DDS |                              5.3.1              |         N/A         |
```

Comment:
Maybe we should add a note that we would like to update to version 6 if the time allows to fix the remaining issue (ros2/rmw_connext#351).